MyException.h

```diff
@@ -1,12 +1,14 @@
-#include <iostream>
-using namespace std;
+#ifndef MYEXCEPTION_H
+#define MYEXCEPTION_H
```

Comment:
missing header guard. this project didn't compile without this change

Reply:
Header guard added to MyException.h